Resources/Prototypes/Entities/Objects/Weapons/Melee/sword.yml

```
@@ -48,18 +48,24 @@
   - type: Sprite
     sprite: DeltaV/Objects/Weapons/Melee/katana.rsi #DeltaV
     state: icon
+  - type: Clothing
+    sprite: DeltaV/Objects/Weapons/Melee/katana.rsi #DeltaV and LPP!
```

yamllint

[warning] too few spaces before comment

(comments)